packages/glibc.rb

```diff
@@ -82,8 +90,10 @@ def self.build
         rootsbindir=#{CREW_PREFIX}/bin
       EOF
 
-      system build_env.transform_keys(&:to_s), '../configure', *config_opts, no_preload_hacks: true
-      system "make PARALLELMFLAGS='-j #{CREW_NPROC}'", no_preload_hacks: true
+      system build_env, '../configure', *config_opts, no_preload_hacks: true
+      # Without using Kernel.system make segfaults.
+      Kernel.system build_env, "make PARALLELMFLAGS='-j #{CREW_NPROC}'", exception: false
+      Kernel.system 'make -j1 || make -j1 || make -j1 || make -j1' if $CHILD_STATUS.exitstatus != 0
```

Comment:
This should be no longer necessary with the new `crew-preload.so`

Reply:
Yeah not needed. Just finishing some tweaks on this... I'm building this on the older glibc containers though because the build still breaks with preload enabled on the older containers...